wmn-data.json

```diff
@@ -515,6 +515,19 @@
       ],
       "cat": "gaming"
     },
+    {
+      "name": "Arsmate",
+      "uri_check": "https://arsmate.com/{account}",
+      "e_code": 200,
+      "e_string": "og:title",
```

Comment:
This is a little too generic as this might be there in non-profile pages. Please take another look at the page and see if you can pull out a section that is only there when a profile is. Something like the `var profile_id = ` element.